kyo-prelude/shared/src/main/scala/kyo/kernel/Context.scala

```diff
@@ -16,11 +16,14 @@ object Context:
         inline def contains[A, E <: ContextEffect[A]](tag: Tag[E]): Boolean =
             context.contains(tag.erased)
 
+        inline def inherit: Context =
+            context.filterNot(_._1 <:< Tag[ContextEffect.Isolated])
```

Comment:
To generalize this approach a bit more, do you think `ContextEffect` should have an `onFork` (or equivalent) function? It seems like this could be useful for other context effects.

'def onFork' can essentially become `identity | default` in this case.

Reply:
Ah. This seems like it could be a limitation in the future... I'll think about this problem more.

Reply:
It seems a reasonable given that the alternative requires code to cache instances in effects like `Abort`, which also requires `asInstanceOf`. I think we should avoid changing core aspects like this one before 1.0.